spec/support/shared_csv_export_behavior.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+shared_examples 'a CSV export' do
```

Comment:
Let's rename the shared examples so that they describe the behavior more clearly and read better in the spec output: 
```ruby
shared_examples 'a page that can export records'
```

Reply:
Renamed